lib/server.go

```diff
@@ -171,7 +166,11 @@ type ServerConfiguration struct {
 }
 
 func NewServer(configuration ServerConfiguration, handler RTRServerEventHandler, simpleHandler RTREventHandler) *Server {
-	sessid := GenerateSessionId()
+	sessids := make([]uint16, 0, int(configuration.ProtocolVersion) + 1)
+	s := GenerateSessionId()
+	for i := 0; i <= int(configuration.ProtocolVersion); i++ {
+		sessids = append(sessids, s + uint16(100 * i))
```

Comment:
can't this overflow if `s` for example is `0xFFFF`? I guess this is an explicit property of Go, right? https://stackoverflow.com/questions/34704843/on-purpose-int-overflow/34704898#34704898

Reply:
The overflow is handled as expected since we use uint16 as type for all session ids variables.
When you run the code below the output is:
65280
65380
65480
44
144
244

Code
```
package main

import "fmt"

func main() {
	s := uint16(0xff00)
	for i := 0; i < 6; i++ {
		n := s + uint16(100 * i)
		fmt.Println(n)
	}
}
```